package.json

```diff
@@ -15,7 +15,8 @@
   },
   "dependencies": {
     "react": "^18.3.1",
-    "react-dom": "^18.3.1"
+    "react-dom": "^18.3.1",
+    "react-router-dom": "^6.26.2"
```

Comment:
Remove this po

Reply:
ang react-router-dom

Reply:
aight ill remove it directly sa github nlng haha

Reply:
done po